.eslintrc

```diff
@@ -0,0 +1,151 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 9,
+        "ecmaFeatures": {
+          "jsx": false
+        }
```

Comment:
We don't actually use React here, do we?

```suggestion
```

Reply:
Nope, I guess adding a false is overkill :)